sales/src/query.rs

```diff
@@ -18,12 +18,13 @@ where
     pub fn active_primary_sales(&self, deps: &Deps, env: Env) -> StdResult<QueryResp> {
         let primary_sales = self.primary_sales.load(deps.storage)?;
         for sale in primary_sales {
-            if (!sale.disabled && sale.end_time.gt(&env.block.time))
-                || sale.end_time.gt(&env.block.time)
+            if sale.disabled {
+                continue
+            } else if (sale.start_time.le(&env.block.time)) && sale.end_time.gt(&env.block.time)
```

Comment:
One comment on this: is there a reason that the end time is not inclusive with GTE rather than just GT here?

Reply:
No, updated